packages/db/src/meta/pouch/adapters/indexeddb.ts

```diff
@@ -16,21 +12,21 @@ export interface DatabasesSettings {
 }
 
 export const getDefaultSettings: GetDefaultSettings = () => ({
-  directory: path.join(Config.getTruffleDataDirectory(), ".db", "sqlite")
+  directory: path.join(Config.getTruffleDataDirectory(), ".db")
 });
 
 export class Databases<C extends Collections> extends Base.Databases<C> {
   private directory: string;
 
   setup(settings: DatabasesSettings) {
-    this.directory = settings.directory;
+    // ensure db files reside in a path that ends with indexeddb
+    // whether specified in config, or using default
+    this.directory = path.join(settings.directory, "indexeddb");
```

Comment:
NOTE: This is to makes sure that the new default persistence location (indexeddb) does not collide with previous default location (sqlite)

Reply:
Why did this get moved here from line 15?

Reply:
This is to make sure that the persistence file path for the default `indexeddb` will always end in `indexeddb` to prevent collision with earlier versions that rely solely on the default path at line 15. This should hold true for user-level, or per-project configs. Should I update the comment?